addons/core/translations/resources/en-us.yaml

```diff
@@ -362,3 +362,9 @@ managed-group:
     welcome:
       title: Welcome to Managed Groups
       description: Create and manage managed groups for an auth method.
+    member:
+      title: Welcome to Accounts
+      description: No accounts in this managed group.
+member:
+  title: Member
+  title_plural: Members
```

Comment:
Thoughts on nesting this under managed-group? `target` resource is a good example of this.

Reply:
Definitely including this, makes all the sense :)